contracts/crowdsale/PausableCrowdsale.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.4.18;
+
+import "./Crowdsale.sol";
+import "../lifecycle/Pausable.sol";
+
+
+/**
+ * @title PausableCrowdsale
+ * @dev Extension of Crowdsale contract that can be paused and unpaused by the pauser role.
```

Comment:
Maybe we should add a comment indicating only _purchases_ are paused (e.g. refunds can still go through, if the crowdsale is never unpaused and misses its goal).